quizapp/src/main/java/billennium/quizapp/controller/CandidateController.java

```diff
@@ -30,7 +31,7 @@ public String getUserId(@PathVariable("email") String email) {
 
     @PostMapping(path = SAVE_CANDIDATE)
     @ResponseStatus(HttpStatus.CREATED)
-    public Candidate saveUser(@RequestBody String email) {
+    public Candidate saveUser(@RequestBody CreateCandidateDto email) {
         return candidateService.save(email);
```

Comment:
i jezeli tu byś miał `candidateService.save(dto.getEmail())` to nie musiałbyś zmieniać nic w serwisie, i importować obiektu CreateCandidateDto  do tesow itp też byś nie musiał

Reply:
racja :D